### ISSUE TITLE
It is not compatible with NC25

### ISSUE THREAD

**Issue by anguschang007** [e-alfred/ocdownloader#250]
![Screenshot at Oct 25 21-04-13](https://user-images.githubusercontent.com/36773065/197781979-09eb6120-f0ac-4b9e-9510-f98917de2748.png)
The app is not compatible with NC25. The right side section is missing!

**Comment by Albirew** [comment]
On my side, i get an error 500-like, and an exception in logs
`Exception: OC\Files\Utils\Scanner::__construct(): Argument #4 ($logger) must be of type Psr\Log\LoggerInterface, OC\Log given, called in /path-to-nxc/apps/ocdownloader/controller/index.php on line 249 in file '/path-to-nxc/lib/private/Files/Utils/Scanner.php' line 95`

**Comment by anguschang007** [comment]
> On my side, i get an error 500-like, and an exception in logs `Exception: OC\Files\Utils\Scanner::__construct(): Argument #4 ($logger) must be of type Psr\Log\LoggerInterface, OC\Log given, called in /path-to-nxc/apps/ocdownloader/controller/index.php on line 249 in file '/path-to-nxc/lib/private/Files/Utils/Scanner.php' line 95`

https://github.com/e-alfred/ocdownloader/pull/247

**Comment by weeman1337** [comment]
Yes. There is not event proper support for 24 yet, although a ready PR exists #247 .There was [a commit](https://github.com/e-alfred/ocdownloader/commit/8a736a6792c65130e7991dd3b073a019f2a2dd66) updating the app info without migrating the rest.  I am removing myself as a collaborator @e-alfred . The reason is that I don't want to put time into this project if the work is ignored :frowning_face:

**Comment by MasterBean7** [comment]
Thanks for point it out @firebird76 
In fact, on my version NC25.0.1 it looks better if the <dav> is removed completely.
However, the automatic reload still creates "glitch" effect.
And workaround #247 was needed.

**Comment by p-bo** [comment]
Just in case it wasn't noticed - Nextcloud improved policy regarding changes in core to ease app developers in maintaining compatibility with newer Nextcloud versions:

https://help.nextcloud.com/t/new-process-for-documenting-core-code-changes-that-affect-app-developers/149828?u=p-bo

**Comment by 31337-4554551n** [comment]
I am also saddened this app isn't getting love. Commenting to get updates if this is ever resolved. This app was one of the reasons I love nextcloud.
@weeman1337 Are you aware of any forks perhaps? Any interest in creating your own? :P

**Comment by freechelmi** [comment]
NcDownloader is a good candidate https://apps.nextcloud.com/apps/ncdownloader

**Comment by 31337-4554551n** [comment]
> NcDownloader is a good candidate https://apps.nextcloud.com/apps/ncdownloader

You're an absolute legend! Had no idea this existed.

None of these have an android companion app though, do they?

**Comment by e-alfred** [comment]
Due to personal reasons, I did not have as much time during the last months for maintaining all the Nextcloud apps I used to do for several years now. 

One mistake was to release of a bogus version without the required fix that should have been packaged. I am very sorry for that, but keeping up was not as easy as it was before. I hope I can do more releases in the future again.

**Comment by NTenic-Hadrev** [comment]
Just in case someone wants to try it. NCDownloader is not working well.
